.gitattributes

```diff
@@ -1,6 +1,8 @@
 # Set the default behavior to LF, because checkstyle enforces it
 * text eol=lf
 
+*.bat eol=crlf
```

Comment:
This pull request was accidentally closed. I then pushed my PR changes, which included the Gradle upgrade, and re-opened the PR.  Could this be why they are showing?  Maybe it did a diff from the closed PR code to the new re-opened PR code.